lexer/tokenizer.go

```diff
@@ -49,9 +48,7 @@ func belongsTogether(current []rune, next rune) bool {
 		switch current[0] {
 		case '+':
 			return next == '+'
-		case '!':
-			fallthrough
-		case '=':
+		case '!', '=':
```

Comment:
`fallthrough` is useful if you have additional logic for the next case, but in this case simple just write comma separated values